src/components/MainCategoryTap.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import { TEXT_DETAILS } from '@assets/constant/constant'
+
+//이 컴포넌트를 호출할때는 위로와 축하로 나눠질 상황에서 실행될 함수들을 꼭 props로 전달해주셔야 합니다.
+type MainCategory = {
+  onRecordCelebration: () => void
+  onRecordConsolation: () => void
+  currentRecordType: string
+}
+function MainCategoryTap({
+  onRecordCelebration,
+  onRecordConsolation,
+  currentRecordType,
+}: MainCategory) {
+  const { CELEBRATION, CONSOLATION } = TEXT_DETAILS
+
+  const typeConfig = {
+    active: 'text-primary-2 border-b-2 border-current pb-4',
+    inactive: 'text-gray-600',
+  }
+
+  const handleRecordTypeToCelebration = (): void => {
+    onRecordCelebration()
+  }
+
+  const handleRecordTypeToConsolation = (): void => {
+    onRecordConsolation()
+  }
+
+  return (
+    <div className="mt-6 flex justify-between border-b border-gray-300 px-10  text-lg font-semibold">
+      <div
+        className={`${
+          currentRecordType === CELEBRATION
+            ? typeConfig.active
+            : typeConfig.inactive
+        }`}
+        onClick={handleRecordTypeToCelebration}
```

Comment:
```suggestion
        onClick={() => setRecordType(CELEBRATION)}
```

세인트 말에 추가로 이렇게 한번에 할 수도 있을 것 같아요!

Reply:
아 저도 동의합니다 굳이 함수로 뺄필요도 없죠 참